src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="ticket-status BC-Gov-15px-black-text">
+    <div>Request submitted date: <span>{{dispute.citizenSubmittedDate | date : 'EEEE, MMMM d, y'}}</span></div>
+    <div>Status: <span>{{dispute.status}}</span></div>
+    <div>Assigned to: <span>{{dispute.jjAssigned}}</span></div>
```

Comment:
On the staff portal dashboard, when jjAssigned is blank, null etc. they want us to display 'Unassigned'.

Reply:
Done. Defaulted value to "Unassigned".